test/profiling/allocations/AllocTestPublisher.cpp

```diff
@@ -27,21 +27,20 @@
 #include <fastdds/dds/publisher/qos/DataWriterQos.hpp>
 #include <fastdds/dds/publisher/qos/PublisherQos.hpp>
 #include <fastdds/dds/topic/qos/TopicQos.hpp>
-#include <fastrtps/types/TypesBase.h>
 
 #include "AllocTestCommon.h"
 #include "AllocTestTypePubSubTypes.h"
 
 using namespace eprosima::fastdds::dds;
 
 #define CHECK_RETURN_CODE(ret) \
-    if (ReturnCode_t::RETCODE_OK != ret) \
+    if (RETCODE_OK != ret) \
     { \
         return false; \
     }
 
 #define CHECK_ENTITY_CREATION(entity) \
-    if (nullptr != entity) \
+    if (nullptr == entity) \
```

Comment:
How could the old code be working?

Reply:
No idea, I was receiving an instantaneous return after executing when I tried to run it.